### ISSUE TITLE
Some Errors spawning in logs for  3 days running

### ISSUE THREAD

**Issue by a4501150** [azerothcore/azerothcore-wotlk#1062]
##### SMALL DESCRIPTION:

Here is a DB Error log generated during 3 days with a server with some players.

##### EXPECTED BLIZZLIKE BEHAVIOUR:

If I found fix I would post here. And if you have fix, please post it here,

##### CURRENT BEHAVIOUR:

```
2018-10-20 12:26:08 SmartScript::ProcessAction: Entry 25514 SourceType 0, Event 0, Link Event 1 not found or invalid, skipped.
2018-10-20 17:45:20 SmartScript::ProcessAction: Entry 25514 SourceType 0, Event 0, Link Event 1 not found or invalid, skipped.
2018-10-20 21:07:41 SmartScript::ProcessAction: Entry 24077 SourceType 0, Event 0, Link Event 1 not found or invalid, skipped.
2018-10-21 15:14:15 SmartScript::ProcessAction: Entry 26670 SourceType 0, Event 18, Link Event 19 not found or invalid, skipped.
2018-10-22 17:55:33 SmartScript::ProcessAction: Entry 30945 SourceType 0, Event 3, Link Event 4 not found or invalid, skipped.




2018-10-20 13:41:17 CreatureTextMgr: Could not find TextGroup 1 for Creature(Bone Warrior) GuidLow 97670 Entry 26126. Ignoring.
2018-10-21 15:23:31 CreatureTextMgr: Could not find TextGroup 4 for Creature(Grauf) GuidLow 2029953 Entry 26893. Ignoring.

2018-10-22 06:41:08 CreatureTextMgr: Could not find Text for Creature(Bloodtalon Scythemaw) Entry 3123 in 'creature_text' table. Ignoring.
2018-10-22 15:00:46 CreatureTextMgr: Could not find Text for Creature(Ebon Blade Prisoner) Entry 30186 in 'creature_text' table. Ignoring.
2018-10-22 17:43:27 CreatureTextMgr: Could not find Text for Creature(Ebon Blade Prisoner) Entry 30194 in 'creature_text' table. Ignoring.
2018-10-22 18:47:42 CreatureTextMgr: Could not find Text for Creature(Bloodtalon Scythemaw) Entry 3123 in 'creature_text' table. Ignoring.
2018-10-22 21:28:38 CreatureTextMgr: Could not find Text for Creature(Bloodworm) Entry 28017 in 'creature_text' table. Ignoring.



2018-10-21 21:09:18 WaypointMovementGenerator::LoadPath: creature Bile Golem (Entry: 28201 GUID: 2032203) doesn't have waypoint path id: 0
2018-10-21 21:09:18 WaypointMovementGenerator::LoadPath: creature Crypt Fiend (Entry: 27734 GUID: 2032204) doesn't have waypoint path id: 0
2018-10-21 21:09:18 WaypointMovementGenerator::LoadPath: creature Devouring Ghoul (Entry: 28249 GUID: 2032205) doesn't have waypoint path id: 0
2018-10-21 21:09:18 WaypointMovementGenerator::LoadPath: creature Crypt Fiend (Entry: 27734 GUID: 2032245) doesn't have waypoint path id: 0
2018-10-21 21:09:18 WaypointMovementGenerator::LoadPath: creature Devouring Ghoul (Entry: 28249 GUID: 2032246) doesn't have waypoint path id: 0
2018-10-21 21:09:18 WaypointMovementGenerator::LoadPath: creature Patchwork Construct (Entry: 27736 GUID: 2032247) doesn't have waypoint path id: 0
2018-10-20 14:05:44 WaypointMovementGenerator::LoadPath: creature Prince Keleseth (Entry: 23953 GUID: 2027980) doesn't have waypoint path id: 0
```

##### BRANCH(ES):

master

**Comment by BarbzYHOOL** [comment]
Missing stuff from the sniffs I guess.

For missing text, must either investigate which text and create it, or remove the need for text, or check if TC has fixed it.

Same for each in fact.

For smart_scripts, it may be that a wrong "link" has been set, so just need to change the column to 0 (but it could also mean the linked row is missing)

**Comment by poszer** [comment]
@a4501150 On what OS, Windows, right?

**Comment by a4501150** [comment]
@poszer yes, it on windows. But I thought the same thing would happen in Linux.

**Comment by Viste** [comment]
When server run more then week with 20+ player you see much more errors with that type

**Comment by Pondaveia** [comment]
Can someone confirm if this issue is still happening?

**Comment by poszer** [comment]
Why closing?

**Comment by Pondaveia** [comment]
Lacking of Hash, also can't reproduce issue.

**Comment by poszer** [comment]
> Lacking of Hash, also can't reproduce issue.

Create server, and let it to run for more than 15/20 days. Than you will reproduce it

**Comment by poszer** [comment]
CreatureTextMgr: Could not find TextGroup 1 for Creature(Bone Warrior) GuidLow 97670 Entry 26126. Ignoring.

This creatures is spawned on wrong place >.< On every combat, we have db errors about TextGroup. And if you take a look around, he doesn't belong to be there. 
https://image.prntscr.com/image/feQG7V4WSs2NGO_4YEZbrA.png

**Comment by Stoabrogga** [comment]
All PRs merged (#1804, #2022, #2023, #2041, #2047, #2052), so this can be finally closed.
